polkadot/zombienet-sdk-tests/tests/disabling/slashing.rs

```diff
@@ -0,0 +1,172 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Test past-session slashing when a malicious validator backs an invalid
+//! candidate and a dispute concluding in a future session. We achieve that by
+//! making some of the honest nodes go offline.
+
+use anyhow::anyhow;
+
+use crate::helpers::{assert_blocks_are_being_finalized, assert_para_throughput};
+use polkadot_primitives::{BlockNumber, CandidateHash, DisputeState, Id as ParaId, SessionIndex};
+use serde_json::json;
+use subxt::{OnlineClient, PolkadotConfig};
+use zombienet_sdk::NetworkConfigBuilder;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn dispute_past_session_slashing() -> Result<(), anyhow::Error> {
+	let _ = env_logger::try_init_from_env(
+		env_logger::Env::default().filter_or(env_logger::DEFAULT_FILTER_ENV, "info"),
+	);
+
+	let images = zombienet_sdk::environment::get_images_from_env();
+
+	let config = NetworkConfigBuilder::new()
+		.with_relaychain(|r| {
+			r.with_chain("westend-local")
+				.with_default_command("polkadot")
+				.with_default_image(images.polkadot.as_str())
+				.with_default_args(vec![
+					"--no-hardware-benchmarks".into(),
+					"-lparachain=debug,runtime=debug".into(),
+				])
+				.with_genesis_overrides(json!({
+					"patch": {
+						"configuration": {
+							"config": {
+								"scheduler_params": {
+									"group_rotation_frequency": 3,
+									"max_validators_per_core": 1,
+									"lookahead": 2,
+									"max_candidate_depth": 3,
+									"allowed_ancestry_len": 2
```

Comment:
I think the parameters should be modified a little:
- We are in the process of bumping `lookahead` to 3 on Kusama (done) and Polkadot (will be done soon). The etst networks also use lookahead=3 (https://github.com/paritytech/polkadot-sdk/pull/7252). I think it's better to set it to 3 here too.
- With https://github.com/paritytech/polkadot-sdk/pull/7254 we shouldn't need `max_candidate_depth` and `allowed_ancestry_len`.

Also in the old test `group_rotation_frequency` was 2 but I believe you did it on purpose?

Reply:
>  Also in the old test group_rotation_frequency was 2 but I believe you did it on purpose?

Bumped it to make sure malus doesn't miss it's backing slot(s) due to flakyness/slowness in CI setup. It needs to be low enough OTOH that malus gets assigned quickly.